packages/fluentui/react-northstar/src/themes/teams/components/Menu/menuItemIndicatorStyles.ts

```diff
@@ -26,8 +26,9 @@ const menuItemIndicatorStyles: ComponentSlotStylesPrepared<MenuItemIndicatorStyl
       content: '" "',
       display: 'block',
       overflow: 'hidden',
-      height: pxToRem(16),
+      height: pxToRem(14),
```

Comment:
@layershifter @mnajdova Before the element being float wasn't affecting the height of the container, now I removed the float and reduced the height to avoid visual regression, this affects only the `ComposeExample` but I think we are ok with it because it was overriding the height/width in the example

Reply:
A bit skeptical about this change, as all icons are 16x16, so it looks awkward why this should require 14 for avoiding visual regressions. Why I am mentioning this is, if the clients override this to use some different icon, we may end up with clipped icon

Reply:
We could leave it has it was and accept the menu item will have 2 px more of height. What do you think?